policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -278,10 +340,26 @@ impl Index {
         Ok(watch.watch.subscribe())
     }
 
-    pub fn lookup_service(&self, addr: IpAddr) -> Option<ServiceRef> {
+    pub fn lookup_service(&self, addr: IpAddr) -> Option<ResourceRef> {
```

Comment:
we have a thread elsewhere above, discussing renaming fields/types. we should call this `lookup_resource` to be consistent with that renaming

Reply:
But this... does not simply looks up any resource. It looks up a service, which is a resource.. And in order to refer to this service returns a `ResourceRef`.  What sort of renaming would make that clearer here?

Reply:
✔️ resolved in another thread